
<!DOCTYPE html>
<html lang='en'>
<head>
    <meta http-equiv="Content-Type" content="text/html; charset=UTF-8"/>
    <title>Tree - fedora-kde/kde6dev-rpmspecs - Pagure.io</title>
        <link rel="shortcut icon" type="image/vnd.microsoft.icon"
        href="/theme/static/favicon.ico?version=5.13.3"/>
    <link href="/theme/static/fedora-bootstrap-1.3.0/fedora-bootstrap.min.css?version=5.13.3"
        type="text/css" rel="stylesheet" />
    <link href="/theme/static/fonts/fonts.css?version=5.13.3"
        rel="stylesheet" type="text/css" />
    <link href="/theme/static/fonts/hack_fonts/css/hack-extended.min.css?version=5.13.3"
        type="text/css" rel="stylesheet" />
    <link href="/theme/static/theme.css?version=5.13.3"
        type="text/css" rel="stylesheet" />

    <link type="text/css" rel="stylesheet" nonce="VpxBxD8eXLi3xa7mO4XUT3xsU" href="/static/vendor/font-awesome/font-awesome.css?version=5.13.3"/>
    <link type="text/css" rel="stylesheet" nonce="VpxBxD8eXLi3xa7mO4XUT3xsU" href="/static/pagure.css?version=5.13.3"/>
<link nonce="VpxBxD8eXLi3xa7mO4XUT3xsU" rel="stylesheet" href="/static/vendor/highlight.js/styles/github.css?version=5.13.3"/>
<link nonce="VpxBxD8eXLi3xa7mO4XUT3xsU" rel="stylesheet" href="/static/vendor/highlightjs-line-numbers/highlightjs-line-numbers.min.css?version=5.13.3"/>
<style nonce="VpxBxD8eXLi3xa7mO4XUT3xsU">
  .hljs {
    background: #fff;
  }
</style>
  </head>
  <body id="home">
    
    <!-- start masthead -->
    <nav class="navbar navbar-light masthead p-0 navbar-expand">
      <div class="container">
        <a href="/" class="navbar-brand">
        <img height="40" src="/theme/static/pagure-logo.png?version=5.13.3"
             alt="pagure Logo" id="pagureLogo"/>
        </a>
        <ul class="navbar-nav ml-auto">

          

          <li class="nav-item">
            <a class="btn btn-primary" href="/login/?next=https://pagure.io/fedora-kde/kde6dev-rpmspecs/blob/master/f/kio-extras-kf6/find-kactivities-stats.patch">Log In</a>
          </li>
        </ul>
      </div>
    </nav>
    <!-- close masthead-->

    <div class="bodycontent">


<div class="bg-light border border-bottom pt-3">
  <div class="container">
    <div class="row mb-3">
      <div class="col-6">
        <div class="row">
        <div class="col-auto pr-0">
            <h3>
<i class="fa fa-calendar-o fa-rotate-270 text-muted"></i></h3>
        </div>
        <div class="col-auto pl-2">
            <h3 class="mb-0">
<a href="/projects/fedora-kde/%2A">fedora-kde</a>&nbsp;/&nbsp;<a href="/fedora-kde/kde6dev-rpmspecs"><strong>kde6dev-rpmspecs</strong></a>
            </h3>
        </div>
        </div>
      </div>
      <div class="col-6 text-right">
        <div class="btn-group">
        <div class="btn-group">
        <a href="#"
            class="btn btn-sm dropdown-toggle btn-outline-primary"
            data-toggle="dropdown" id="watch-button">
          <i class="fa fa-clone fa-fw"></i>
          <span>Clone</span>
        </a>
        <div class="dropdown-menu dropdown-menu-right">
          <div class="m-3" id="source-dropdown">
            <div>
              <h5><strong>Source Code</strong></h5>

              <div class="form-group">
                <div class="input-group input-group-sm">
                  <div class="input-group-prepend"><span class="input-group-text">GIT</span></div>
                  <input class="form-control bg-white select-on-focus" type="text" value="https://pagure.io/fedora-kde/kde6dev-rpmspecs.git" readonly>
                </div>
              </div>
            </div>
          </div>
        </div>

      </div>
    </div>
  </div>
</div>

<ul class="nav nav-tabs nav-small border-bottom-0">
  <li class="nav-item mr-2 text-dark">
    <a class="nav-link active" href="/fedora-kde/kde6dev-rpmspecs">
        <i class="fa fa-code fa-fw text-muted"></i>
        <span class="d-none d-md-inline">Source</span>
    </a>
  </li>

    <li class="nav-item mr-2 text-dark">
        <a class="nav-link" href="/fedora-kde/kde6dev-rpmspecs/issues">
            <i class="fa fa-fw text-muted fa-exclamation-circle"></i>
            <span class="d-none d-md-inline">Issues&nbsp;</span>
            <span class="badge badge-secondary py-0 d-none d-md-inline">
              0
            </span>
        </a>
    </li>

    <li class="nav-item mr-2 text-dark">
      <a class="nav-link" href="/fedora-kde/kde6dev-rpmspecs/pull-requests">
          <i class="fa fa-fw text-muted fa-arrow-circle-down"></i>
          <span class="d-none d-md-inline">Pull Requests&nbsp;</span>
          <span class="badge badge-secondary py-0 d-none d-md-inline">
            0
          </span>
      </a>
    </li>



    <li class="nav-item mr-2 text-dark">
      <a class="nav-link" href="/fedora-kde/kde6dev-rpmspecs/stats">
          <i class="fa fa-line-chart fa-fw text-muted"></i>
          <span class="d-none d-md-inline">Stats</span>
      </a>
    </li>


</ul>
  </div>
</div>

<div class="container pt-5 repo-body-container">
  <div class="row">
    <div class="col-2">
<nav class="nav nav-tabs nav-sidetabs flex-column">
  <a class=
      "nav-link nowrap
"
      href="/fedora-kde/kde6dev-rpmspecs">
      <i class="fa fa-home text-muted fa-fw"></i>&nbsp;<span class="d-none d-md-inline">Overview</span>
  </a>
  <a class=
    "nav-link nowrap
 active"
    href="/fedora-kde/kde6dev-rpmspecs/tree/master">
    <i class="fa fa-file-code-o text-muted fa-fw"></i>&nbsp;Files
  </a>
  <a class=
    "nav-link nowrap
"
    href="/fedora-kde/kde6dev-rpmspecs/commits/master">
    <i class="fa fa-list-alt text-muted fa-fw" data-glyph="spreadsheet"></i>&nbsp;Commits
  </a>
  <a class=
    "nav-link nowrap
"
    href="/fedora-kde/kde6dev-rpmspecs/branches?branchname=master">
    <i class="fa fa-random text-muted fa-fw"></i>&nbsp;Branches
  </a>
  <a class=
    "nav-link nowrap
"
    href="/fedora-kde/kde6dev-rpmspecs/forks">
    <i class="fa fa-code-fork text-muted fa-fw"></i>&nbsp;Forks
  </a>
  <a class=
    "nav-link nowrap
"
    href="/fedora-kde/kde6dev-rpmspecs/releases">
    <i class="fa fa-tags text-muted fa-fw"></i>&nbsp;Releases
  </a>
</nav>    </div>
    <div class="col-10">
  <div class="row mb-1">
    <div class="col-sm-6">
    <h3>
      Files
    </h3>
    </div>

    <div class="col-sm-6">
      <div class="float-right">
          <div class="btn-group">
            <a href="#" class="btn btn-outline-light border-secondary text-dark btn-sm dropdown-toggle"
                    data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
                    <span class="fa fa-random fa-fw"></span> Branch: <span class="font-weight-bold">master</span>
          </a>
            <div class="dropdown-menu dropdown-menu-right">
                  <a class="dropdown-item pl-1 active" href="/fedora-kde/kde6dev-rpmspecs/tree/master">
                      <span class="fa fa-random fa-fw"></span> <span class="font-weight-bold">master</span>
                  </a>
            </div>
          </div>
    </div>
  </div>
  </div>
    <div class="card mb-3">
      <div class="card-header">
        <ol class="breadcrumb p-0 bg-transparent mb-0">
          <li class="breadcrumb-item">
            <a href="/fedora-kde/kde6dev-rpmspecs/tree/master">
              <span class="fa fa-random">
              </span>&nbsp; master
            </a>
          </li>
<li class="breadcrumb-item"><a href="/fedora-kde/kde6dev-rpmspecs/blob/master/f/kio-extras-kf6">
            <span class="fa fa-folder"></span>&nbsp; kio-extras-kf6</a>
          </li>
          <li class="active breadcrumb-item">
            <span class="fa fa-file" data-glyph="">
            </span>&nbsp; find-kactivities-stats.patch
          </li>
        </ol>
      </div>

  <div class="card-body p-0">
            <div class="bg-light border text-right pr-2">
                <form class="btn btn-sm" method="POST" name="fork_project"
                    action="/fork_edit/fedora-kde/kde6dev-rpmspecs/edit/master/f/kio-extras-kf6/find-kactivities-stats.patch">
                    <button class="btn btn-sm btn-secondary fork_project_btn">
                            Fork and Edit
                    </button>
                    
                </form>

                <a class="btn btn-secondary btn-sm" href="/fedora-kde/kde6dev-rpmspecs/blob/master/f/kio-extras-kf6/find-kactivities-stats.patch" title="View as blob">Blob</a>

                <a class="btn btn-secondary btn-sm" href="/fedora-kde/kde6dev-rpmspecs/blame/kio-extras-kf6/find-kactivities-stats.patch?identifier=master" title="View git blame">Blame</a>

                <a class="btn btn-secondary btn-sm" href="/fedora-kde/kde6dev-rpmspecs/history/kio-extras-kf6/find-kactivities-stats.patch?identifier=master" title="View git log for this file">History</a>

                <a class="btn btn-secondary btn-sm" href="/fedora-kde/kde6dev-rpmspecs/raw/master/f/kio-extras-kf6/find-kactivities-stats.patch" title="View as raw">Raw</a>
            </div>

        <pre class="syntaxhighlightblock"><code class="lang-diff">diff --git a/CMakeLists.txt b/CMakeLists.txt
index 8153b01..7d20433 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -79,7 +79,7 @@ set_package_properties(Qt6Sql PROPERTIES
     TYPE OPTIONAL
 )
 
-find_package(KF6ActivitiesStats ${KF_MIN_VERSION} QUIET)
+find_package(KF6ActivitiesStats QUIET)
 set_package_properties(KF6ActivitiesStats PROPERTIES
     PURPOSE &#34;Provides the recentlyused:/ KIO worker&#34;
     TYPE OPTIONAL
</code></pre>
  </div>
 </div> <!-- end .card-->

</div>
</div>
</div>
    </div>

        <div class="footer pt-4 text-white">
        <div class="container">
            <div class="d-flex align-items-center">
                <div>
                    <div>Powered by <a href="https://pagure.io/pagure" class="notblue">Pagure</a> 5.13.3</div>
                    <div>
                        <a href="https://docs.pagure.org/pagure/usage/index.html" class="notblue">Documentation</a> &bull;
                        <a href="https://pagure.io/pagure/new_issue" class="notblue">File an Issue</a> &bull;
                        <a href="/about">About</a> &bull;
                        <a href="/ssh_info" class="notblue">SSH Hostkey/Fingerprint</a>
                    </div>
                </div>
                <div class="ml-auto text-right">
                    <div>&copy; Red Hat, Inc. and others.</div>
                </div>
            </div>
        </div>
    </div>


    <script type="text/javascript" nonce="VpxBxD8eXLi3xa7mO4XUT3xsU" src="/static/vendor/jquery/jquery.min.js?version=5.13.3"></script>

    <script src="/static/vendor/bootstrap/bootstrap.bundle.min.js?version=5.13.3"></script>

    <script type="text/javascript" nonce="VpxBxD8eXLi3xa7mO4XUT3xsU">
      $('[data-toggle="tooltip"]').tooltip({placement : 'bottom'});
      $(".cancel_btn").click(function() {
        history.back();
      });
    </script>

<script type="text/javascript"  nonce="VpxBxD8eXLi3xa7mO4XUT3xsU" src="/static/vendor/lazyload/lazyload.min.js?version=5.13.3"></script>

<script type="text/javascript" nonce="VpxBxD8eXLi3xa7mO4XUT3xsU">
window.addEventListener("load", function(event) {
    lazyload();
});
</script>

<script type="text/javascript" nonce="VpxBxD8eXLi3xa7mO4XUT3xsU">
$("#giturl-toggle").on('click', function(event){
  event.stopPropagation();
  $("#giturl-more").toggle();
  $("#giturl-toggle").hide();
})

$(".fork_project_btn").click(function() {
  $('#fork_project').submit();
});

$(".select-on-focus").on("focus", function() {
  $(this).select();
});

</script>

<script type="text/javascript" nonce="VpxBxD8eXLi3xa7mO4XUT3xsU" src="/static/vendor/highlight.js/highlight.pack.js?version=5.13.3"></script>
<script type="text/javascript" nonce="VpxBxD8eXLi3xa7mO4XUT3xsU" src="/static/vendor/highlightjs-line-numbers/highlightjs-line-numbers.min.js?version=5.13.3"></script>
<script type="text/javascript" nonce="VpxBxD8eXLi3xa7mO4XUT3xsU" src="/static/vendor/highlight.js/spec.js?version=5.13.3"></script>

<script type="text/javascript" nonce="VpxBxD8eXLi3xa7mO4XUT3xsU">
  $(document).ready(function() {
  $('.fork_project_btn').click($("[name=fork_project]").submit);

  $('pre.syntaxhighlightblock code').each(function(i, block) {
    hljs.highlightBlock(block);
    hljs.lineNumbersBlock(block);
  });

  var cls = "highlighted-line";
  var lines = location.hash.substr(2).split('-').map(function (x) { return parseInt(x, 10) });
  if (! isNaN(lines[0]))
  {
    for (var i = lines[lines.length - 1]; i >= lines[0]; i--) {
      $('#_' + i).parent().parent().addClass(cls);
    }
    setTimeout(function(){
      $("#_" + lines[0]).get(0).scrollIntoView({behavior: "instant", block: "start", inline: "nearest"});
    }, 50);
  }
});
</script>

<script type="text/javascript" nonce="VpxBxD8eXLi3xa7mO4XUT3xsU">

  function updateHighlight() {
    var cls = "highlighted-line";
    $('.' + cls).removeClass(cls)
    if (location.hash !== '') {
      var lines = location.hash.substr(2).split('-').map(function (x) { return parseInt(x, 10) });
      for (var i = lines[lines.length - 1]; i >= lines[0]; i--) {
        $('[data-line-number=' + i + ']').closest('tr').addClass(cls);
      }
      return lines;
    }
    return [];
  }
  $(window).on('hashchange', updateHighlight);
  var selected = [];
  $("[data-line-number]").click(function (ev) {
    var line = $(this).attr('data-line-number');
    if (ev.shiftKey) {
      selected = selected.slice(-1).concat(line);
    } else {
      selected = [line];
    }

    var hash = '_' + selected[0];
    if (selected.length === 2) {
      hash = '_' + Math.min(selected[0], selected[1]) + '-' + Math.max(selected[0], selected[1]);
    }
    window.location.hash = hash;
    return false;
  });

</script>


</body>
</html>